<ui version="4.0" >
 <class>window</class>
 <widget class="QMainWindow" name="window" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>274</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize" >
   <size>
    <width>470</width>
    <height>274</height>
   </size>
  </property>
  <property name="maximumSize" >
   <size>
    <width>470</width>
    <height>274</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Process Watch Dog</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="pwatchdog.qrc" >
    <normaloff>:/images/images/icon.png</normaloff>:/images/images/icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <widget class="Line" name="line" >
    <property name="geometry" >
     <rect>
      <x>186</x>
      <y>20</y>
      <width>20</width>
      <height>221</height>
     </rect>
    </property>
    <property name="orientation" >
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>10</y>
      <width>171</width>
      <height>231</height>
     </rect>
    </property>
    <property name="font" >
     <font>
      <family>Verdana</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="title" >
     <string>Watched Processes</string>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout" >
     <item>
      <widget class="QListWidget" name="procListWidget" >
       <property name="frameShape" >
        <enum>QFrame::Box</enum>
       </property>
       <property name="horizontalScrollBarPolicy" >
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_2" >
    <property name="geometry" >
     <rect>
      <x>210</x>
      <y>10</y>
      <width>251</width>
      <height>231</height>
     </rect>
    </property>
    <property name="font" >
     <font>
      <family>Verdana</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="title" >
     <string>Current Process</string>
    </property>
    <widget class="QToolButton" name="addButton" >
     <property name="geometry" >
      <rect>
       <x>120</x>
       <y>60</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="toolTip" >
      <string>New Process</string>
     </property>
     <property name="text" >
      <string>...</string>
     </property>
     <property name="icon" >
      <iconset resource="pwatchdog.qrc" >
       <normaloff>:/images/images/new.png</normaloff>:/images/images/new.png</iconset>
     </property>
     <property name="iconSize" >
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
    </widget>
    <widget class="QLineEdit" name="procIdText" >
     <property name="geometry" >
      <rect>
       <x>120</x>
       <y>30</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QToolButton" name="remButton" >
     <property name="geometry" >
      <rect>
       <x>160</x>
       <y>60</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="toolTip" >
      <string>Attach Process</string>
     </property>
     <property name="text" >
      <string>...</string>
     </property>
     <property name="icon" >
      <iconset resource="pwatchdog.qrc" >
       <normaloff>:/images/images/remove.png</normaloff>:/images/images/remove.png</iconset>
     </property>
     <property name="iconSize" >
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
    </widget>
    <widget class="QLabel" name="label_4" >
     <property name="geometry" >
      <rect>
       <x>20</x>
       <y>30</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font" >
      <font>
       <family>Verdana</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text" >
      <string>Process ID:</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_3" >
     <property name="geometry" >
      <rect>
       <x>30</x>
       <y>100</y>
       <width>201</width>
       <height>121</height>
      </rect>
     </property>
     <property name="title" >
      <string>Status</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2" >
      <item>
       <widget class="QToolButton" name="toolButton" >
        <property name="enabled" >
         <bool>true</bool>
        </property>
        <property name="minimumSize" >
         <size>
          <width>80</width>
          <height>80</height>
         </size>
        </property>
        <property name="maximumSize" >
         <size>
          <width>80</width>
          <height>80</height>
         </size>
        </property>
        <property name="text" >
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <zorder>addButton</zorder>
    <zorder>procIdText</zorder>
    <zorder>remButton</zorder>
    <zorder>label_4</zorder>
    <zorder>detachButton</zorder>
    <zorder>groupBox_3</zorder>
    <zorder>toolButton</zorder>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>470</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar" />
 </widget>
 <resources>
  <include location="pwatchdog.qrc" />
 </resources>
 <connections/>
</ui>
